src/Test.UnitTests.Sarif.Multitool.Library/ApplyPolicyCommandTests.cs

```diff
@@ -0,0 +1,88 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using FluentAssertions;
+using Moq;
+using System;
+using System.IO;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Microsoft.CodeAnalysis.Sarif.Multitool
+{
+    public class ApplyPolicyCommandTests
+    {
+        private static readonly ResourceExtractor Extractor = new ResourceExtractor(typeof(ApplyPolicyCommandTests));
+
+        [Fact]
+        public void WhenOutputFormatOptionsAreInconsistent_Fails()
+        {
+            const string InputFilePath = "AnyFile.sarif";
+
+            var mockFileSystem = new Mock<IFileSystem>();
+            mockFileSystem.Setup(_ => _.FileExists(InputFilePath)).Returns(true);
+
+            var options = new ApplyPolicyOptions
+            {
+                InputFilePath = InputFilePath,
+                Inline = true,
+                PrettyPrint = true,
+                Minify = true
+            };
+
+            int returnCode = new ApplyPolicyCommand(mockFileSystem.Object).Run(options);
+
+            returnCode.Should().Be(1);
+
+            mockFileSystem.Verify(_ => _.FileExists(InputFilePath), Times.Once);
+            mockFileSystem.VerifyNoOtherCalls();
+        }
+
+        [Fact]
+        public void WhenInputContainsOnePolicy_ShouldSucceed()
+        {
+            string path = "WithPolicy.sarif";
+            File.WriteAllText(path, Extractor.GetResourceText($"ApplyPolicyCommand.{path}"));
+
+            // Verify log loads, has correct Result count, and spot check a Result
+            SarifLog log = ExecuteTest(path);
+            log.Runs[0].Results.Count.Should().Be(1);
+            log.Runs[0].Results[0].Level.Should().Be(FailureLevel.Error);
+
+            File.Delete(path);
+        }
```

Comment:
No need to delete the files. They are written to the test binaries directory and overwritten on each run (and deleted on a clean build).